demos/common/python/models/ultra_lightweight_face_detection.py

```diff
@@ -58,6 +59,10 @@ def preprocess(self, inputs):
         resized_image = resize_image(image, (self.w, self.h))
         meta = {'original_shape': image.shape,
                 'resized_shape': resized_image.shape}
+        if self.input_transform.reverse_input_channels:
+            resized_image = cv2.cvtColor(resized_image, cv2.COLOR_BGR2RGB)
+        resized_image = (resized_image - self.input_transform.mean_values) / self.input_transform.scale_values
```

Comment:
This code is repeated everywhere; it would make sense to move it to the `InputTransform` class itself (e.g. `resized_image = self.input_transform.apply(resized_image)`).